extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts

```diff
@@ -859,22 +859,12 @@ export class BdcRouterApi {
      * @param connection
      * @param {*} [options] Override http request options.
      */
-    public getCluster (xRequestId: string, connection: string, options: any = {}) : Promise<{ response: http.IncomingMessage; body: any;  }> {
+    public getCluster (xRequestId?: string, connection?: string, options: any = {}) : Promise<{ response: http.IncomingMessage; body: any;  }> {
         const localVarPath = this.basePath + '/api/v1/bdc/';
         let localVarQueryParameters: any = {};
         let localVarHeaderParams: any = (<any>Object).assign({}, this.defaultHeaders);
         let localVarFormParams: any = {};
 
-        // verify required parameter 'xRequestId' is not null or undefined
-        if (xRequestId === null || xRequestId === undefined) {
-            throw new Error('Required parameter xRequestId was null or undefined when calling getCluster.');
-        }
-
-        // verify required parameter 'connection' is not null or undefined
-        if (connection === null || connection === undefined) {
-            throw new Error('Required parameter connection was null or undefined when calling getCluster.');
-        }
-
         localVarHeaderParams['X-Request-Id'] = ObjectSerializer.serialize(xRequestId, "string");
```

Comment:
what happens in objectserializer if xRequestId or connection is null/undefined?

Reply:
The header is empty. 

This is probably a result of a bug in the swagger spec where this was defined as being required but actually isn't (we fixed a number of these early on in the BDC development so this isn't a new thing). Calling it without these headers doesn't have any negative effect.